CVE-2022-3105.yml

```diff
@@ -0,0 +1,371 @@
+CVE: CVE-2022-3105
+CWE:
+  - 476
+ipc:
+  note: This is primarily used for communication between user-space applications and kernel-space InfiniBand (IB) and Remote Direct Memory Access (RDMA) drivers. It doesn't directly deal with inter-process communication (IPC) mechanisms such as OS signals, pipes, stdin/stdout, message passing, clipboard, or file-based IPC.
```

Comment:
I think you can focus on what the vulnerability does and you don't need to reiterate IPC because we should all know what it is at this point